packages/@react-aria/test-utils/src/events.ts

```diff
@@ -11,9 +11,17 @@
  */
 
 import {act, fireEvent} from '@testing-library/react';
+import {isMac} from '@react-aria/utils';
 import {UserOpts} from './types';
 
 export const DEFAULT_LONG_PRESS_TIME = 500;
+export function getAltKey(): 'Alt' | 'ControlLeft' {
```

Comment:
? what does `isMac` return in a test environment?
I feel like we should hook this into the user config if we need it

Reply:
it checks windows.navigator if it exists and then sees if it contains Mac, so if the user hasn't mocked it in their test suite it will return false. I flip flopped a bit on whether or not to include this as something the user defines or to just check it directly via `isMac` and have settled on the latter since it should match the internal logic of our press hooks/etc which use the same `isMac` call. Additionally, trying to mock this for the user means we will need them to tell us to how mock things in general for their test environment which I feel we handle in the testing docs anyways.